image/image_test.go

```diff
@@ -699,3 +714,123 @@ func (s *imageSuite) TestNewToolingStoreWithAuth(c *C) {
 	c.Check(user.StoreMacaroon, Equals, "MACAROON")
 	c.Check(user.StoreDischarges, DeepEquals, []string{"DISCHARGE"})
 }
+
+func (s *imageSuite) TestBootstrapToRootDirLocalSnapsWithStoreAsserts(c *C) {
+	restore := image.MockTrusted(s.storeSigning.Trusted)
+	defer restore()
+
+	rootdir := filepath.Join(c.MkDir(), "imageroot")
+
+	// FIXME: bootstrapToRootDir needs an unpacked gadget yaml
```

Comment:
Not sure what the FIXME means here as the code below seems to unpack the gadget snap.